package.json

```diff
@@ -153,7 +160,8 @@
       "react-app",
       "plugin:prettier/recommended",
       "plugin:jsx-a11y/recommended",
-      "prettier/react"
+      "prettier/react",
```

Comment:
Does `npm start` interperet the typescript code, or do you have to build and start?

Reply:
I.E. does `react-scripts start` work with typescript?

Reply:
Similarly, do you have to specify anything special w/ running `npm build`.